eliot-service/eliot/health_resource.py

```diff
@@ -0,0 +1,110 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""
+Application-health related Falcon resources.
+"""
+
+from collections import OrderedDict
+import json
+
+from dockerflow.version import get_version
+import falcon
+import markus
+
+
+METRICS = markus.get_metrics(__name__)
+
+
+class BrokenResource:
+    """Handle ``/__broken__`` endpoint."""
+
+    def on_get(self, req, resp):
+        """Implement GET HTTP request."""
+        METRICS.incr("broken.count")
+        # This is intentional breakage
+        raise Exception("intentional exception")
```

Comment:
We need to make sure that nginx is set up to protect this endpoint behind basic auth. I'll add that to the project plan.